ft-sdk/src/from_request/mod.rs

```diff
@@ -40,6 +48,12 @@ impl FromRequest for ft_sdk::Connection {
     }
 }
 
+impl FromRawRequest for ft_sdk::Connection {
+    fn from_request(_req: &http::Request<bytes::Bytes>) -> Result<Self, ft_sdk::Error> {
```

Comment:
Not fond of this approach. Lets create our own type, so we can do: 

```rs
fn from_request(_req: &http::Request<ft_sys::Body>) -> Result<Self, ft_sdk::Error> {}
```

And inside body we store `Bytes`. And we have a private `.json` which is a refcell option serde_json::value, and we compute it the first time it is accessed via public `.json()`.